### ISSUE TITLE
macOS 11 kernel panic

### ISSUE THREAD

**Issue by sanarena** [kubernetes/minikube#9103]
minikube start --driver=hyperkit --kubernetes-version v1.18.3 --disk-size 60g --memory 3500

**Steps to reproduce the issue:** 

(Worth to mention I am running macOS v11 beta 5)
1. Upgrade minikube v.1.11.0 to v1.12.3
2. Run minikube
3. Kernel panic appear after a minute of running.

**Full output of failed command:** 
All commands were successful.

**Full output of `minikube start` command used, if not already included:**
All commands were successful.

**Optional: Full output of `minikube logs` command:**
No chance to get the logs because of kernel panic.

</details>

Meanwhile, I downgraded to v1.11.0, and the issue is solved.

**Comment by sanarena** [comment]
Issue happened after I downgraded to v.11.0
Wondering what might have caused it.
It was working well for past few weeks before I initiate the upgrade via brew.

**Comment by sanarena** [comment]
Just to add context to this issue, I figured out kernel panic was because of nfsd in macos 11 beta 5.
We are using nfs to mount some volumes from host into containers, as nfs was fastest method we found.
However, macos 11 beta 5 have a bug, where if nfs volume get "read", it will kernel panic.
I tested it by connecting to NFS in this machine from another machine and got kernel panic. 
So this issue was not related to minikube at all, but a big sur bug.
Downgraded to macos 11 beta 4, and updated minikube to latest version, everything is working fine.
